shell/browser/api/atom_api_web_contents.cc

```diff
@@ -2035,7 +2035,7 @@ void WebContents::StartDrag(const mate::Dictionary& item,
     base::MessageLoopCurrent::ScopedNestableTaskAllower allow;
     DragFileItems(files, icon->image(), web_contents()->GetNativeView());
   } else {
-    args->ThrowError("Must specify either 'file' or 'files' option");
+    args->ThrowError("Must specify either 'file' or 'files' parameter.");
```

Comment:
```suggestion
    args->ThrowError("startDrag() parameter 'item' must have a nonempty property 'file'.");
```

Note: I removed `'files'` from the error message here because `file` is the only documented parameter and it can be a string or array. `files` is either undocumented or was removed.